pkgs/tools/security/scilla/default.nix

```
@@ -5,16 +5,25 @@
 
 buildGoModule rec {
   pname = "scilla";
-  version = "1.2.4";
+  version = "1.2.7";
 
   src = fetchFromGitHub {
     owner = "edoardottt";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-8ZRYgQ4xME71vlO0nKnxiCqeju0G4SwgEXnUol1jQxk=";
+    hash = "sha256-gZuNXQaxHJYLsEaOpNYo7ybg3f0GhkpiaLrex5lkDu4=";
   };
 
-  vendorSha256 = "sha256-Y4Zi0Hy6ydGxLTohgJGF3L9O+79z+3t+4ZA64otCJpE=";
+  vendorHash = "sha256-bVGmleuOJzi/Sz7MJlnQuJsDgRWuwieLUx8hcyKkWXI=";
+
+  checkFlags =
+    let
+      skippedTests = [
+        # Network access required
+        "TestIPToHostname"
+      ];
+    in
+    [ "-skip" "${builtins.concatStringsSep "|" skippedTests}" ];
```

this seems a bit unnecessary

```suggestion
  checkFlags = [
    # requires network access
    "-skip=TestIPToHostname"
  ];
```

---

This will make it easier to add more test cases.